go/gin/go.mod

```diff
@@ -1,11 +1,5 @@
 module main
 
-require (
-	github.com/gin-contrib/sse v0.0.0-20190301062529-5545eab6dad3 // indirect
-	github.com/gin-gonic/gin v1.3.0
-	github.com/golang/protobuf v1.3.0 // indirect
-	github.com/mattn/go-isatty v0.0.6 // indirect
-	github.com/ugorji/go/codec v0.0.0-20190204201341-e444a5086c43 // indirect
-	gopkg.in/go-playground/validator.v8 v8.18.2 // indirect
-	gopkg.in/yaml.v2 v2.2.2 // indirect
-)
+go 1.12
```

Comment:
You can explicitly define the `go` **version** ?

Reply:
Yes. you can try the following command for initiating the project.

```
$ go mod init xxxx
```